ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -1147,7 +1147,7 @@ protected Node visitInsertInto(final InsertInto node, final Object context) {
   protected Node visitDropTable(final DropTable node, final Object context) {
     return node.getLocation()
         .map(location ->
-            new DropTable(node.getLocation().get(),
+            new DropTable(node.getLocation(),
```

Comment:
surely, this style of method can not be simplified to:

```
protected Node visitDropTable(final DropTable node, final Object context) {
    return new DropTable(node.getLocation(),
                         node.getTableName(),
                         node.getIfExists(),
                         node.isDeleteTopic());
  }
```

That was kinda the whole point of making the constructors taking `Optional` public! lol.

Reply:
Yes, missed this part. Updated it for all the ones that have public constructor now.